west.yml

```diff
@@ -74,7 +74,7 @@ manifest:
       revision: 3776c158fe138a72c97c187e4d31c81c37884be9
       path: modules/crypto/mbedtls
     - name: mcuboot
-      revision: c2bd757332bfde11c27019f937b0c61d5e9b9fd9
+      revision: pull/2/head
```

Comment:
Please replace with e96b6873bd8a64185796ac47b941f56dd29b6e3f, now that the required fix is merged